.github/workflows/issues_and_pull_requests.yml

```
@@ -31,7 +31,7 @@ jobs:
   handle_stale:
     name: Handle stale state
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-slim
```

```suggestion
    runs-on: ubuntu-latest
```

CI is failing on slim